sys/Makefile.dep

```diff
@@ -559,6 +559,12 @@ ifneq (,$(filter asymcute,$(USEMODULE)))
   USEMODULE += event_callback
 endif
 
+ifneq (,$(filter event_periodic_callback,$(USEMODULE)))
+  USEMODULE += event_callback
+  USEMODULE += event_periodic
+  USEMODULE += event_thread
```

Comment:
What is this module needed for? From the code I don't see any obvious reason for this dependency.

Reply:
Which of those modules do you mean?

Reply:
`event_thread`. The module that's in the line I commented on...

Reply:
We need an event thread to execute the callbacks.

Reply:
Yes, but that's application code (you provide the queue to the init function). An application might as well just use its own runner thread (which should remain an option anyways IMHO).